recipe/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "momepy" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/momepy-{{ version }}.tar.gz
+  sha256: 413ed9221dc4b2190621f8cb37789f09ea882f7e81dfbffd5731a22da6e61bab
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [py<310 or s390x]
+
+requirements:
+  host:
+    - python
+    - setuptools >=61.0
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python
+    - geopandas >=0.12.0
+    - libpysal >=4.6.0
+    - networkx >=2.7
+    - packaging
+    - pandas !=1.5.0
+    - shapely >=2
+    - tqdm >=4.63.0
+    - inequality
+    - jupyter
+    - mapclassify
+    - matplotlib
+    - networkx
+    - numba
+    - esda
```

Comment:
```suggestion
```
I'm not seeing where any of these are used.

Reply:
I found these in https://github.com/pysal/momepy/blob/main/environment.yml. I was also having local test failures without some of these

Reply:
Ahh, I see. Thanks! Well I'm not sure of the best practice in this situation, but I thought the `environment.yml` dependencies could be ignored. However, if they make the tests failed, then maybe they would do better in the `test` section since they don't look like `run` dependencies.

Reply:
Ok let me check that out! Thank you.

Reply:
Yes it would fit better in test/require